### PR TITLE
macOS: Fix WebView print hang

### PR DESCRIPTION
**Pull request by TcT2k**
Fixes: [#19063](https://trac.wxwidgets.org/ticket/19063)

### CODE REVIEW

File: src/osx/webview_webkit.mm [hunk 1]
```diff
@@ -271,7 +271,8 @@ - (id)initWithHandler:(wxWebViewHandler*) handler;
         [op setShowsProgressPanel: false];
     }
     // Print it.
-    [op runOperation];
+	[op runOperationModalForWindow:m_webView.window
```
Comment (lanurmi) [inline src/osx/webview_webkit.mm:274]: Wrong indentation here?

Comment (TcT2k) [reply src/osx/webview_webkit.mm]: Fixed

**Comment by vadz** [comment]
Thanks a lot for fixing this!

**Comment by whalley** [comment]
Have tested this and it does indeed work. Thanks!

**Pull request closed without merging**